lib/assets/data_transform_cell/main.js

```diff
@@ -8,6 +8,9 @@ export async function init(ctx, payload) {
   ctx.importCSS(
     "https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap"
   );
+  ctx.importCSS(
+    "https://cdn.jsdelivr.net/npm/remixicon@3.2.0/fonts/remixicon.min.css"
```

Comment:
Shouldn't we keep on using the SVG ones instead of loading all of remix icons? 🤔

Reply:
It’s your call!

Reply:
Any preferences @jonatanklosko?

Reply:
@cristineguadelupe told me we use the remix icons everywhere else, so let's merge it.